querydsl-core/src/main/java/com/querydsl/core/types/dsl/ComparableExpression.java

```diff
@@ -387,4 +387,58 @@ public BooleanExpression loeAny(SubQueryExpression<? extends T> right) {
         return Expressions.comparableOperation(getType(), Ops.AggOps.MAX_AGG, mixin);
     }
 
+    /**
+     * Create a {@code nullif(this, other)} expression
+     *
+     * @param other
+     * @return nullif(this, other)
+     */
+    @Override
+    public ComparableExpression<T> nullif(Expression<T> other) {
+        return Expressions.comparableOperation(this.getType(), Ops.NULLIF, mixin, other);
+    }
+
+    /**
+     * Create a {@code nullif(this, other)} expression
+     *
+     * @param other
+     * @return nullif(this, other)
+     */
+    @Override
+    public ComparableExpression<T> nullif(T other) {
+        return nullif(ConstantImpl.create(other));
+    }
+
+    /**
+     * Create a {@code coalesce(this, exprs...)} expression
+     *
+     * @param exprs additional arguments
+     * @return coalesce
+     */
+    @Override
+    @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
Is this necessary? What about `Expression<T>... exprs`?
That applies to the whole inheritance hierarchy.

Reply:
Good suggestion, I'll check if it introduces other binary incompatibilities and otherwise apply the patch.